File: keybind-common/src/main/java/io/github/solusmods/eternalcore/keybind/EternalCoreKeybind.java
package io.github.solusmods.eternalcore.keybind;

import dev.architectury.platform.Platform;
import dev.architectury.utils.Env;

public class EternalCoreKeybind {
    public static void init() {
        if (Platform.getEnvironment() == Env.CLIENT) {
            EternalCoreKeybindClient.init();
        }
    }
}

File: keybind-common/src/main/kotlin/io/github/solusmods/eternalcore/keybind/EternalCoreKeybind.kt
package io.github.solusmods.eternalcore.keybind

import dev.architectury.platform.Platform
import dev.architectury.utils.Env

object EternalCoreKeybind {
    @JvmStatic
    fun init() {
        if (Platform.getEnvironment() == Env.CLIENT) {
            EternalCoreKeybindClient.init()
        }
    }
}
